tasks/nginx.yml

```diff
@@ -3,12 +3,22 @@
 - name: Copy Nginx config for Jitsi Meet domain.
   template:
     dest: "/etc/nginx/sites-available/{{ jitsi_meet_server_name }}.conf"
-    src: nginx.conf.j2
+    src: nginx/sites-available/vhost.conf.j2
     owner: root
     group: root
     mode: "0644"
   notify: restart nginx
 
+- name: Copy Nginx multiplexer config if local turnserver is used.
+  template:
+    dest: "/etc/nginx/modules-enabled/60-jitsi-meet.conf"
```

Comment:
Should this be a write to `modules-available` and then a symlink?

Reply:
Thanks for the review, @ramondelafuente! I decided to implement it exactly the way the Debian package does it in order to keep the delta as small as possible. That's why I didn't do the symlink dance here ;)